server/api/mentions_handler.py

```diff
@@ -0,0 +1,40 @@
+import os
+from datetime import datetime
+from pymongo import MongoClient
+from bson import json_util
+import json
+from flask import Blueprint, request, jsonify
+mentions_handler = Blueprint('mentions_handler', __name__)
+
+
+@mentions_handler.route('/mentions', methods=['GET', 'POST', 'DELETE'])
+def mentions():
+    connection_string = 'mongodb+srv://{username}:{password}@{server}/?retryWrites=true&w=majority'
+    client = MongoClient(connection_string.format(
+        username=os.getenv('USERNAME'),
+        password=os.getenv('PASSWORD'),
+        server=os.getenv('DB_SERVER')
+    ))
+    db = client[os.getenv('DB_NAME')]
```

Comment:
I think for now this is fine. but I'm hoping we can keep all direct db related operates in the db package. Keep separation of concerns and all that. I'm currently working on a number of functions that would handle operations like this for us.

Reply:
so once I get that approved this would transform into something like `db.interface.addMention(mention)`

Reply:
Sounds great - maybe the best thing to do is once your PR is merged I can make the appropriate updates to this one then merge it in

Reply:
Might be easier to have your stuff come through first. Then I can just swap our your code for my function call once I get that in. I'm thinking of having Mentions be an actual object, make it easier for everyone to read.